src/components/Search/index.tsx

```diff
@@ -52,12 +52,13 @@ const SearchInputPanel = styled.input`
     width: 100%;
     padding-left: 10px;
     padding-right: 20px;
+    border: 1px solid #606060;
     border-radius: 6px;
   }
   padding-left: 20px;
   padding-right: 50px;
   border-width: 0px;
-  border-radius: 6px 0 0 6px;
+  border-radius: 6px;
```

Comment:
PC search bar is left of testnet flag, so why set all corners `border-radius` to `6px`?

Reply:
OK.
I originally wanted to fix this issue, now i optimized it.
![image](https://user-images.githubusercontent.com/23251844/60581804-26af2080-9dba-11e9-8981-b5fc8b1061c4.png)